winpr/libwinpr/smartcard/smartcard_pcsc.c

```diff
@@ -981,11 +980,13 @@ static LONG WINAPI PCSC_SCardListReadersW(SCARDCONTEXT hContext, LPCWSTR mszGrou
 			return SCARD_E_NO_MEMORY;
 	}
 
+	status =
+	    PCSC_SCardListReaders_Internal(hContext, mszGroupsA, (LPSTR*)&mszReadersA, pcchReaders);
 	if (status == SCARD_S_SUCCESS)
```

Comment:
I must test, but I'd bet that it doesn't work as the caller expect a list of wstrings

Reply:
@hardening yes, and the `UTF8` list is converted to a `WCHAR` list a few lines below